Fix the aml pkgconfig dependency

The correct dependency is 'aml1' within the Yocto Project.  See neatvnc.

Upstream-Status: Inappropriate [OpenEmbedded Specific]

Signed-off-by: Mark Hatle <mark.hatle@amd.com>

--- a/libweston/backend-vnc/meson.build.orig	2026-06-10 09:11:29.027864648 -0600
+++ b/libweston/backend-vnc/meson.build	2026-06-10 09:11:57.464193950 -0600
@@ -8,7 +8,7 @@
 	error('VNC backend requires neatvnc which was not found. Or, you can use \'-Dbackend-vnc=false\'.')
 endif
 
-dep_aml = dependency('aml', version: ['>= 0.3.0', '< 0.4.0'], required: false, fallback: ['aml', 'aml_dep'])
+dep_aml = dependency('aml1', version: ['>= 1.0.0'], required: false, fallback: ['aml', 'aml_dep'])
 if not dep_aml.found()
 	error('VNC backend requires libaml which was not found. Or, you can use \'-Dbackend-vnc=false\'.')
 endif
